### PR TITLE
Don't document SCC for operator flow

### PR DESCRIPTION
**Pull request by booxter**
Operator already creates the SCC rules needed.

### CODE REVIEW

File: administration/intro.adoc [hunk 1]
```diff
@@ -178,26 +178,21 @@ Using command line interface
...
-$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-privileged
+$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-handler
 $ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-controller
 $ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-apiserver
 ----
 
-- When you use the operator flow:
-[source,bash]
-----
-$ oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-operator
```
Comment (slintes) [inline administration/intro.adoc]: To my best knowledge this is still needed. The operator can't add the kubevirt-* accounts to the privileged SCC without being in that SCC itself.

Comment (booxter) [reply administration/intro.adoc]: I've deployed the operator on OCP4 several days ago and it didn't need the SCC part, but I will recheck.

**Comment by booxter** [comment]
Not sure why it seemed to work the first attempt but on redeployment with the latest installer it now requires SCC workaround. Do this PR is invalid.

**Pull request closed without merging**